### DISCUSSION TITLE
Canonical hit object naming

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#20887] [Q&A]
In lazer we have

| osu! | osu!taiko | osu!catch | osu!mania |
| :-: | :-: | :-: | :-: |
| hit circle | (strong) hits (don, kat) | fruits | note |
| slider (head, tail) | drumroll | juice stream (large droplet (`Droplet`), small droplet (`TinyDroplet`)[^drop]) | hold note (head, tail) |
| spinners | swell (swell tick[^s-tick]) | banana (shower) |  |
|  |  | hyperfruit (name never displayed) |  |

common names in the community

| osu! | osu!taiko | osu!catch | osu!mania |
| :-: | :-: | :-: | :-: |
| hit circle / circle | (small/large) note (don, kat) | fruit | note / rice |
| slider (head, end) | drumroll / drum roll | slider trail ((juice) drop, droplet) | hold note / long note (start, end) |
| spinner | denden/shaker | banana (shower) |  |
|  |  | hyperfruit |  |

[^drop]: the results screen shows "LARGE DROPLET" and "TINY DROPLET"
[^s-tick]: called "S TICK" in the results screen for some reason, even though there would be space for "SWELL TICK"

continuing from [discussion on discord](https://discord.com/channels/188630481301012481/188630652340404224/1033709522511986718), i would like to put into question the naming of "hits" in taiko and to a lesser degree "large droplets" and "small droplets" in catch, and also spark discussion for what each object should canonically be called.

**Comment by peppy** [comment]
> 2. called "S TICK" in the results screen for some reason, even though there would be space for "SWELL TICK" [↩](#user-content-fnref-s-tick-976353b8d7576bc2926fe48fbc9aca35)

"s tick" is short for "small tick" for whatever that's worth (not saying it's any better than a proposed alternative).
